vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -38,12 +47,20 @@ public String listSpaces(@PathVariable(required = false) String page,
         model.addAttribute("totalPages", imageService.getTotalPages());
         model.addAttribute("currentPageNumber", pageNo);
         model.addAttribute("totalImageCount", imageService.getTotalImageCount());
-        model.addAttribute("images", imageService.getImages(pageNo, sortedBy, order));
+        List<IVSImage> images = imageService.getImages(pageNo, sortedBy, order);
+        LinkedHashMap<String, List<ISpace>> imageToSpaces = new LinkedHashMap<String, List<ISpace>>();
```

Comment:
Doubt: Why are you using LinkedHashMap?
 Is maintaining order required?

Reply:
Not required, changed to Map, HashMap.

Reply:
ok!